contrib/epee/include/serialization/wire/adapted/vector.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2022-2023-2023, The Monero Project
+// Copyright (c) 2022-2024-2023, The Monero Project
```

Comment:
```suggestion
// Copyright (c) 2022-2024, The Monero Project
```

Reply:
this file added after the 2023 update had the imperfect data here : 

https://github.com/monero-project/monero/commit/a6ce2dc7c5b31ee85c09a92e2531fb377a5517ad#diff-a09395d06c958b393d1bba70da8182483c2db997643f960a86ac83c1c70b3ff3R1